src/audio_unit/audio_format.rs

```diff
@@ -6,61 +6,191 @@
+    /// AC-3 codec that provides data packaged for transport over an IEC 60958 compliant digital
+    /// audio interface.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    F60958AC3(Option<StandardFlags>),      // = 1667326771,
+    /// Apple's implementation of the IMA 4:1 ADPCM codec.
+    ///
+    /// **Available** in OS X v10.2 and later.
     AppleIMA4,                            // = 1768775988,
-    MPEG4AAC(Option<Mpeg4Object>),        // = 1633772320,
-    MPEG4CELP(Option<Mpeg4Object>),       // = 1667591280,
-    MPEG4HVXC(Option<Mpeg4Object>),       // = 1752594531,
-    MPEG4TwinVQ(Option<Mpeg4Object>),     // = 1953986161,
+    /// MPEG-4 AAC codec.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    MPEG4AAC(Option<Mpeg4ObjectId>),      // = 1633772320,
+    /// MPEG-4 CELP codec.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    MPEG4CELP(Option<Mpeg4ObjectId>),     // = 1667591280,
+    /// MPEG-4 HVXC codec.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    MPEG4HVXC(Option<Mpeg4ObjectId>),     // = 1752594531,
+    /// MPEG-4 TwinVQ codec.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    MPEG4TwinVQ(Option<Mpeg4ObjectId>),   // = 1953986161,
+    /// MACE 3:1.
+    ///
+    /// **Available** in OS X v10.3 and later.
     MACE3,                                // = 1296122675,
+    /// MACE 6:1.
+    ///
+    /// **Available** in OS X v10.3 and later.
     MACE6,                                // = 1296122678,
+    /// μLaw 2:1.
+    ///
+    /// **Available** in OS X v10.3 and later.
     ULaw,                                 // = 1970037111,
+    /// aLaw 2:1.
+    ///
+    /// **Available** in OS X v10.3 and later.
     ALaw,                                 // = 1634492791,
+    /// QDesign Music.
+    ///
+    /// **Available** in OS X v10.3 and later.
     QDesign,                              // = 1363430723,
+    /// QDesign2 Music.
+    ///
+    /// **Available** in OS X v10.3 and later.
     QDesign2,                             // = 1363430706,
+    /// QUALCOMM PureVoice.
+    ///
+    /// **Available** in OS X v10.3 and later.
     QUALCOMM,                             // = 1365470320,
+    /// MPEG-1/2, Layer 1 audio.
+    ///
+    /// **Available** in OS X v10.3 and later.
     MPEGLayer1,                           // = 778924081,
+    /// MPEG-1/2, Layer 2 audio.
+    ///
+    /// **Available** in OS X v10.3 and later.
     MPEGLayer2,                           // = 778924082,
+    /// MPEG-1/2, Layer 3 audio.
+    ///
+    /// **Available** in OS X v10.3 and later.
     MPEGLayer3,                           // = 778924083,
-    TimeCode(Option<AudioTimeStampFlag>), // = 1953066341,
+    /// A stream of IOAudioTimeStamp structures.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    TimeCode(Option<AudioTimeStampFlags>), // = 1953066341,
+    /// A stream of MIDIPAcketList structures where the time stamps in the MIDIPacket structures
```

Comment:
I have no idea HOW I spotted this, but there's a slight capitalization error; should be `MIDIPacketList`, not `MIDIPAcketList`